lib/npm-util.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+const execa = require('execa');
+const pTimeout = require('p-timeout');
+
+exports.connectToNpm = () => pTimeout(
+	(async () => {
+		try {
+			await execa('npm', ['ping']);
+			return true;
+		} catch (_) {
+			throw new Error('Connection to npm registry failed');
+		}
+	})(),
+	15000,
+	'Connection to npm registry timed out'
+);
```

Comment:
Would be more correct to name this something like `checkConnectionToNpm`.

Reply:
Since this method is declared in the context of npm-related utils, wouldn't `checkConnection` suffice?

Reply:
Yes